packages/hooks/use-fills.tsx

```diff
@@ -0,0 +1,71 @@
+import { useConnect, warn } from "frontity";
+import { Fill } from "@frontity/types";
+
+interface FillsPackage {
```

Comment:
Instead of redeclaring the type, you can import `Package`. That way if we change the origin type, it changes here as well.

Reply:
👍